eZ/Bundle/EzPublishIOBundle/Migration/MigrationHandler.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
```

Comment:
@bdunogier Uh, no, it's right below on line 6. Commented on the wrong file, maybe? 😕